src/core/public/rendering/_base.scss

```diff
@@ -35,39 +35,16 @@
   position: relative; // This is temporary for apps that relied on this being present on `.application`
 }
 
-// TODO: This is problematic because it doesn't stay in line with EUI:
-// adapted from euiHeaderAffordForFixed as we need to handle the top banner
-@mixin kbnAffordForHeader($headerHeight) {
-  padding-top: $headerHeight;
-
-  #app-fixed-viewport {
-    top: $headerHeight;
-  }
-
-  .euiFlyout,
-  .euiCollapsibleNav {
-    top: $headerHeight;
-    height: calc(100% - #{$headerHeight});
-  }
-
-  @include euiBreakpoint('m', 'l', 'xl') {
-    .euiPageSideBar--sticky {
-      max-height: calc(100vh - #{$headerHeight});
-      top: #{$headerHeight};
-    }
-  }
-}
-
 .kbnBody {
-  @include kbnAffordForHeader($kbnHeaderOffset);
+  @include euiHeaderAffordForFixed($kbnHeaderOffset);
```

Comment:
I'll assume you know what you're doing here (right 😅 ?)

Reply:
Yeah, I honestly couldn't find a reason why there was a Kibana-specific one since the EUI mixin accepts a custom `$headerHeight` as well and all the classes were targeting the `body` element.

I checked with/without banners, and full screen modes. Though I did just realize I forgot about that `#app-fixed-viewport`. Adding that back in.